app/views/volunteers/index.html.erb

```diff
@@ -1,37 +1,112 @@
-<!--Include links to view volunteers, view homes, search or sort volunteers/homes-->
-<div>
-    <h1>Volunteers</h1>
-    <h2>Welcome Rebuilding Together of the Triangle Staff!</h2>
-    <h3>Below is a list of your volunteers.</h3>
+<style>
+ 
+    .body {
+        margin-top: 250px;
+        margin-left: 25px;
+        margin-right: 25px;
+    }
+    
+    .banner {
+        background: #7ac043;
+    }
+        
+    .logo {
+        margin-top: 20px;
+        margin-left: 40px;
+        margin-bottom: 20px;
+    }
+      
+    .title {
+        color: white;
+        margin-top: 70px;
+        margin-left: 200px;
+        font-family: verdana;
+    }
+    .center_text {
+        text-align: center;
+    }
+    
+    .card_title {
+        color: #7ac043;
+    }
+    
+    .card_padding {
+        padding: 10px;
+    }
+    
+    .picture {
+        width: 100%;
+        height: auto;
+    }
+</style>
+
+<!--Include nav bard for links to view volunteers, view homes, search or sort volunteers/homes-->
+<div class='fixed-top'>
```

Comment:
Consider changing this so the header isn't fixed. On a small screen, it could be difficult to see the actual form, especially if the screen is narrow.

![image](https://user-images.githubusercontent.com/42787645/55346385-5fb8bb80-5480-11e9-8877-fd163a7db06c.png)